runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPanel.java

```diff
@@ -312,6 +313,16 @@ public void mouseEntered(MouseEvent mouseEvent)
 		final JMenuItem reset = new JMenuItem("Reset All");
 		reset.addActionListener(e ->
 		{
+			final int result = JOptionPane.showOptionDialog(overallPanel,
+				"If you are logged in, this action will also permanently delete the current loot from the RuneLite website.",
```

Comment:
This is incorrect. Remote loot is deleted only if sync is enabled

Reply:
Ah, indeed. Thanks!